### PR TITLE
Yoinks Three Eye from Baystation as an admeme chem

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Three Eye is a chem that only comes packaged in pills which overdose the user. The user trips the fuck out, hearing haunting messages until his brain melts.

Also fixes overdose_start, nobody noticed for 2 years?

## Why It's Good For The Game
Bus chem bus chem

## Changelog
:cl:
fix: chems call overdose_start again
add: Three Eye, a mysterious liquid from another realm...
tweak: Changes some flags on plane_masters, as they appear to be inverted due to a bug or something.
/:cl:

### CODE REVIEW

**Review by genessee-forgot-his-password-again — APPROVED** [review]
my name is shitty admin number 17 and i approve this chemical

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -2340,3 +2340,85 @@
...
+		"Come back from there. Please."
+	)
+	var/static/list/overdose_messages = list(
+		"THE SIGNAL THE SIGNAL THE SIGNAL THE SIGNAL",
+		"IT CRIES IT CRIES IT WAITS IT CRIES",
+		"NOT YOURS NOT YOURS NOT YOURS NOT YOURS",
+		"THAT IS NOT FOR YOU",
+		"IT RUNS IT RUNS IT RUNS IT RUNS",
+		"THE BLOOD THE BLOOD THE BLOOD THE BLOOD",
+		"THE LIGHT THE DARK A STAR IN CHAINS"
```
Comment (genessee-forgot-his-password-again) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
		"THE LIGHT THE DARK A STAR IN CHAINS"
		"MOTH FEET FEET FEET"
		"THE VENTS CALL FOR YOU"
		"WE LIVE IN A SOCIETY"
		"DEEPFRY THE KEPORI"
```

comedy (not a maintainer so you don't have to listen to me)

**Comment by zestybastard** [comment]
chemist gaming here. please give it a recipe with wittel or something stupid so i have a chance

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Kapu1178) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: no

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -2340,3 +2340,91 @@
...
+
+/datum/reagent/three_eye/on_mob_end_metabolize(mob/living/L)
+	. = ..()
+	if(L.hallucination >= 50)
+		to_chat(L, "<span class='danger'><font size = 6>Your mind reels and the world begins to fade away...</font></span>")
+		if(iscarbon(L))
+			var/mob/living/carbon/C = L
+			addtimer(CALLBACK(C, /mob/living/carbon.proc/adjustOrganLoss, ORGAN_SLOT_BRAIN, 200), 5 SECONDS) //Deathblow to the brain
+		else
+			addtimer(CALLBACK(L, /mob/living.proc/gib), 5 SECONDS)
```
Comment (ZephyrTFA) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:2455]: Let's not just completely RR people though

Comment (Kapu1178) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2455]: but thats part of the fuuuuun.

Comment (zestybastard) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2455]: gibbing is fixable, but it doesnt make sense for this drug unless you like, psionically tear yourself to bits

Comment (Kapu1178) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2455]: > gibbing is fixable, but it doesnt make sense for this drug unless you like, psionically tear yourself to bits

Gibbing is only for simple mobs, as they do not have brains.

Comment (ZephyrTFA) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm:2455]: Right, doesn't see the carbon check, oops

File: code/_onclick/hud/plane_master.dm [hunk 3]
```diff
@@ -46,7 +46,7 @@
 /atom/movable/screen/plane_master/game_world
 	name = "game world plane master"
 	plane = GAME_PLANE
-	appearance_flags = PLANE_MASTER //should use client color
+	appearance_flags = PLANE_MASTER|NO_CLIENT_COLOR //should use client color
```
Comment (ZephyrTFA) [inline code/_onclick/hud/plane_master.dm:49]: The comment says it should use client color, why are adding the no client color flag?

Comment (Kapu1178) [reply code/_onclick/hud/plane_master.dm:49]: This flag is fucked, and apparently inverted.

**Pull request merged**